### PR TITLE
[Android] Remove enter and exit transition

### PR DESCRIPTION
**Pull request by weihuoya**
this transition is a little cheap. 
don't know this pr is appropriate.
:open_mouth:

### CODE REVIEW

File: Source/Android/jni/MainAndroid.cpp [hunk 1]
```diff
@@ -278,7 +278,7 @@ JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_StopEmulatio
                                                                                   jobject obj)
 {
   std::lock_guard<std::mutex> guard(s_host_identity_lock);
-  Core::SaveScreenShot("thumb", true);
+  //Core::SaveScreenShot("thumb", true);
```
[the remark this answers is not in the document]
Comment (JosJuice) [reply Source/Android/jni/MainAndroid.cpp]: Yes, please don't keep commented-out code.

**Comment by JosJuice** [comment]
@dolphin-emu-bot rebuild

**Comment by JosJuice** [comment]
Seems like a sensible idea to me. These transitions were designed around the idea that screenshots would be shown in the game list, which isn't true anymore.

**Comment by mahdihijazi** [comment]
@JosJuice if that is your concern, we can still switch the animation to use the game covers? but is needed? I have to admit after testing it seeing the black screen for few seconds is annoying but I would care more about the ideal cases where devices suppose to load the game instantly maybe?

**Comment by JosJuice** [comment]
I don't think we need to do an animation with the game covers.

**Pull request merged**